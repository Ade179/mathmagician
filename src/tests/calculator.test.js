import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../components/calculator';

test('should match the snapshot', () => {
  const calc = renderer.create(<Calculator />);
  const tree = calc.toJSON();
  expect(tree).toMatchSnapshot();
});
