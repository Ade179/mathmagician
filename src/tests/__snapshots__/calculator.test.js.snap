// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<section
  className="cont"
>
  <div
    className="cta"
  >
    Let us do some math!
  </div>
  <div
    className="container"
  >
    <div
      className="screen"
    >
      <span
        className="answer"
      >
        0
      </span>
    </div>
    <div
      className="buttons"
    >
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="white span2"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="white"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="orange"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</section>
`;
